packages/atomic/src/components/common/types.ts

```diff
@@ -0,0 +1,238 @@
+export type SearchStatusState = {
```

Comment:
Rather than copying all these types from Headless, perhaps we could re-export them along with their documentation

Reply:
Let's discuss this in scrum. I'm not convinced this is the right approach.

Reply:
For now I re-exported the types from the basic @coveo/headless which are still valid so far...

Reply:
I recall that a lot of those types already have a "generic" version of them, called "Core*". Perhaps we could re-export those.